.github/workflows/sanity-checks.yml

```diff
@@ -41,12 +42,13 @@ jobs:
           RUSTC_WRAPPER: "sccache"
 
   benchmark-check:
+    if: ${{ !github.event.pull_request.draft }}
     # timeout-minutes: 90 # <- Not needed as ubunt-latest is free of charge
     name: bench-check-${{ matrix.runtime }}
     runs-on: ubuntu-latest #-4-cores
     strategy:
       matrix:
-        runtime: [ altair, centrifuge ]
+        runtime: [ centrifuge ]
```

Comment:
It's really not necessary to also check Altair here. If it works for Centrifuge, we have ensured validity. In the worst case, something might be off in the Altair runtime config but we can always just use the Centrifuge weights anyways.

Reply:
Good point regarding the weights! Maybe even make sense to move the weights to `runtime-common` 🤔